stdlib/log-api/src/main/java/org/ballerinalang/stdlib/log/AbstractLogFunction.java

```diff
@@ -80,8 +80,8 @@ public String get() {
         boolean logEnabled = LOG_MANAGER.getPackageLogLevel(pckg).value() <= logLevel.value();
         if (logEnabled) {
```

Comment:
Do we need this variable `logEnabled`? Or can we change this to
```suggestion
         if (LOG_MANAGER.getPackageLogLevel(pckg).value() <= logLevel.value()) {
```

Reply:
Fixed in cbd8988